Messenger/src/test/java/com/epam/testmodule/FileModeTest.java

```diff
@@ -42,10 +42,10 @@ void checkInput_WhenZeroIsProvided_ApplicationShouldExit() {
     Iterable<DynamicTest> dynamicTests_checkFilePath_IfFilePathExist_ShouldReturnTrue() {
         Assertions.assertTrue(this.anotherTempDir.isDirectory(), "Should be a directory ");
         FileMode fileMode = new FileMode();
-        String file2 = "C:\\Users\\Ivan_Pylypiv\\Desktop\\Library\\Messenger\\data2.txt";
-        String file3 = "C:\\Users\\Ivan_Pylypiv\\Desktop\\Library\\Messenger\\data3.txt";
-        String file4 = "C:\\Users\\Ivan_Pylypiv\\Desktop\\Library\\Messenger\\data4.txt";
-        String file5 = "C:\\Users\\Ivan_Pylypiv\\Desktop\\Library\\Messenger\\data5.txt";
+        String file2 = "C:\\Users\\Ivan_Pylypiv\\IdeaProjects\\Library\\Messenger\\data2.txt";
```

Comment:
same here, it would be better to use relative paths to resource files instead of absolute ones